### PR TITLE
Everywhere: Gently remove the ladybird android port

### PR DESCRIPTION
**Pull request by nico**
With Ladybird now being its own repository, there's little reason to keep the Ladybird Android port in the SerenityOS repository.

(The Qt port is useful to be able to test changes to LibWeb in lagom so it'll stay around. Similar for the AppKit port, since getting Qt on macOS is a bit annoying. But if the AppKit port is too much pain to keep working, we should toss that too.

Eventually, the lagom browser ports should move out from Ladybird/ to Meta/Lagom/Contrib, but for now it might make sense to leave them where they are to keep cherry-picks from ladybird easier.)

### CODE REVIEW

**Review by kennethmyhra — APPROVED** [review]

**Comment by ADKaster** [comment]
Didn't catch it in my scroll through but there's a nightly android j GitHub actions ob that should be removed if you missed it

**Comment by ADKaster** [comment]
Also, is removing all android support from the core libraries the right call? I've seen people run the Shell and `js` from inside termux before. which is neat, but also makes sense if we don't actually care :)

**Comment by nico** [comment]
> Also, is removing all android support from the core libraries the right call? I've seen people run the Shell and `js` from inside termux before. which is neat, but also makes sense if we don't actually care :)

I think it's fine to put that back if people actually run lagom binaries on Android, but I figured it's better to err on removing too much than too little.

**Comment by nico** [comment]
> Didn't catch it in my scroll through but there's a nightly android GitHub actions job that should be removed if you missed it

Do I just delete that file? I added a commit for that, thanks. Could you take a look if it looks right?

**Comment by ADKaster** [comment]
yes, just deleting the file is the way. for Azure DevOps there was a 'main' file that triggered the jobs, but for github just having the workflow file in the right place is the thing to do. looks good to me

File: .github/actions/setup/action.yml [hunk 1]
```diff
@@ -81,7 +81,7 @@ runs:
         sudo udevadm trigger --name-match=kvm
```
Comment (kennethmyhra) [inline .github/actions/setup/action.yml:81]: nit, but I think the commit message is supposed to be prefixed with CI:

**Pull request merged**